src/configure.js

```diff
@@ -0,0 +1,16 @@
+let defaults = {
+  adapter    : require('./adapters/ajax'),
+  body       : undefined,
+  params     : undefined,
+  headers    : {},
+  method     : 'GET',
```

Comment:
Yeah, I agree. Like the adapter needs it's own defaults... It's worth thinking that through all the way before this merges.